### PR TITLE
refacturedRuttonSettingsRefactoring

### PR DESCRIPTION
**Pull request by Jorjo77**

### CODE REVIEW

File: lib/screens/home/home_screen.dart [hunk 1]
```diff
@@ -116,3 +115,15 @@ class HomeScreen extends StatelessWidget {
     );
   }
 }
+
+  class ButtonSettings {
```
Comment (HristoPetkov94) [inline lib/screens/home/home_screen.dart:119]: this is never used, delete it or is there reason why we need it ?

**Review by HristoPetkov94 — CHANGES_REQUESTED** [review]

File: lib/screens/home/home_screen.dart (hunk 1 above)
Comment (Jorjo77) [reply lib/screens/home/home_screen.dart:119]: OK, I've confused refactoring ButtonSettings.

**Comment by Jorjo77** [comment]
I've fix it: https://github.com/Cost-Effective-Software-Project-11/gp5/pull/17/commits/30aa6c8b45ce09b623457e024622680f6a81f46c

**Comment by Jorjo77** [comment]
Do I need to make new pool request or when approve this one I'll merge?

**Review by HristoPetkov94 — APPROVED** [review]

**Review by Peshp — APPROVED** [review]

**Pull request merged**